docs/models/outlines.md

```diff
@@ -0,0 +1,85 @@
+# Outlines
+
+## Install
+
+To use [`OutlinesModel`][pydantic_ai.models.OutlinesModel], you need to either install `pydantic-ai`, or install `pydantic-ai-slim` with the `outlines` optional group:
+```bash
+pip/uv-add "pydantic-ai-slim[outlines]"
+```
+
+As Outlines is a library allowing you to run model from various different providers, it does not include the necessary dependencies for any model, but instead requires you to install the optional group for the provider you want to use. To do so, you can install `pydantic-ai-slim` with an optional group composed of outlines, a dash, and the name of the model. For instance:
```

Comment:
Does this mean we should drop the `outlines` optional group entirely?

Reply:
Ah yes, that makes sense